ui/src/sandbox.rs

```diff
@@ -235,7 +230,7 @@ impl Sandbox {
         let version_output = vec_to_str(output.stdout)?;
 
         let mut info: BTreeMap<String, String> = version_output.lines().skip(1).filter_map(|line| {
-            let mut pieces = line.splitn(2, ":").fuse();
+            let mut pieces = line.splitn(2, ':').fuse();
```

Comment:
I really should profile this sometime; I'm almost certain that the string form is faster than the character one, but that's based on gut feeling.

Reply:
Looks like the string version is about 10-15% slower.  I ran both versions of this function through [criterion.rs](https://github.com/japaric/criterion.rs), these were the results:

```
double quote            time:   [1.4755 us 1.4763 us 1.4770 us]           
               
Found 9 outliers among 100 measurements (9.00%)
  3 (3.00%) low mild
  6 (6.00%) high mild

single quote            time:   [1.2746 us 1.2754 us 1.2763 us]                          

Found 7 outliers among 100 measurements (7.00%)
  2 (2.00%) high mild
  5 (5.00%) high severe
```

Reply:
huh. guess the clippy devs did the analysis before doing the lint ;-)